### PR TITLE
Add GPU and CPU example for stablelm-zephyr-3b

### PR DESCRIPTION
**Pull request by JinBridger**

### CODE REVIEW

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/stablelm/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+### 1. Install
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a Python environment for IPEX-LLM:
+```bash
+conda create -n llm python=3.9 # recommend to use Python 3.9
+conda activate llm
+
+pip install --pre --upgrade ipex-llm[all] # install the latest ipex-llm nightly build with 'all' option
+
```
Comment (Oscilloscope98) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/stablelm/README.md:18]: Please add some comments for why we need transformers==4.38.0 here

Maybe refer to here: https://huggingface.co/stabilityai/stablelm-zephyr-3b/blob/8b471c751c0e78cb46cf9f47738dd0eb45392071/config.json#L21

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/stablelm/generate.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+import torch
+import time
+import argparse
+
+from ipex_llm.transformers import AutoModelForCausalLM
+from transformers import AutoTokenizer
+
+# you could tune the prompt based on your own model,
+# here the prompt tuning refers to https://huggingface.co/stabilityai/stablelm-zephyr-3b
```
Comment (Oscilloscope98) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/stablelm/generate.py:25]: https://huggingface.co/stabilityai/stablelm-zephyr-3b#usage

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/stablelm/generate.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    
+    # Generate predicted tokens
+    with torch.inference_mode():
+        prompt = STABLELM_PROMPT_FORMAT.format(prompt=args.prompt)
+        input_ids = tokenizer.encode(prompt, return_tensors="pt")
+        st = time.time()
+        # if your selected model is capable of utilizing previous key/value attentions
+        # to enhance decoding speed, but has `"use_cache": false` in its model config,
+        # it is important to set `use_cache=True` explicitly in the `generate` function
+        # to obtain optimal performance with BigDL-LLM INT4 optimizations
```
Comment (Oscilloscope98) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/stablelm/generate.py:60]: BigDL-LLM -> IPEX-LLM

**Review by Oscilloscope98 — COMMENTED** [review]
Please add the mode [here](https://github.com/intel-analytics/ipex-llm/blob/main/README.md#verified-models) and [here](https://github.com/intel-analytics/ipex-llm/blob/main/docs/readthedocs/source/index.rst#verified-models)

**Review by plusbang — APPROVED** [review]
LGTM

**Review by Oscilloscope98 — APPROVED** [review]

**Pull request merged**